pytorch/inference/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -166,10 +166,12 @@ RUN pip uninstall -y torch \
 
 RUN conda install -y -c conda-forge "pyyaml>=5.4,<5.5"
 RUN pip install \
-    "Pillow>=8.3.2,<8.4" \
-    "awscli<2"
+    "Pillow>=9.0.1" \
+    "awscli<2" \
+    "urllib3>=1.26.5" \
+    mkl
```

Comment:
why is MKL installed here?

Reply:
PyTorch requires MKL: https://github.com/pytorch/pytorch#install-dependencies
It's included. I think the error comes from version mismatch. Is the error still occurring now?